### PR TITLE
Update build settings for 1.14.2 forge

### PR DESCRIPTION
**Pull request by jadar**
### Overview
I am taking a look at what it takes to update the mod to 1.14.2. These are the changes I needed to import the project into IntelliJ. 

### Changes
- Update forge/MC version for 1.14.2.
- Add `mods.toml` file.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -167,14 +167,14 @@ if(hasProperty('mod_core_plugin')) {
...
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = targetCompatibility = compileJava.sourceCompatibility = compileJava.targetCompatibility = '1.8'
 
 compileJava {
     options.debug = true
     options.debugOptions.debugLevel = 'source,lines,vars'
 }
 
+/*
```
Comment (Kobata) [inline build.gradle]: While this stuff isn't important for locally building (in fact it shouldn't really do anything unless you've set up the configuration correctly), it is *quite* important for the CI/release build process so just disabling it isn't really a good idea.

File: src/main/resources/META-INF/mods.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# This is an example mods.toml file. It contains the data relating to the loading mods.
```
Comment (Kobata) [inline src/main/resources/META-INF/mods.toml]: You probably don't need all the example text, especially stuff that's very easily outdated like the comments on what the current versions are.

The short description of each field is fine, though.

File: src/main/resources/META-INF/mods.toml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# The version number of the mod - there's a few well known ${} variables useable here or just hardcode it
+version="${file.jarVersion}" #mandatory
+ # A display name for the mod
+displayName="Inventory Tweaks" #mandatory
+# A URL to query for updates for this mod. See the JSON update specification <here>
+#updateJSONURL="http://myurl.me/" #optional
+# A URL for the "homepage" for this mod, displayed in the mod UI
+#displayURL="http://example.com/" #optional
+# A file name (in the root of the mod JAR) containing a logo for display
+#logoFile="examplemod.png" #optional
```
Comment (Kobata) [inline src/main/resources/META-INF/mods.toml]: This one and the next few are in the current mcmod.info file I believe.

**Review by Kobata — CHANGES_REQUESTED** [review]

**Comment by jadar** [comment]
Let me know if there are any other changes that you would like to see. I think I addressed all your comments, so far. I clicked resolve but I'm not sure if that marks everything as "fixed". I haven't used Github's code review process much before.

**Review by Kobata — APPROVED** [review]

**Comment by safetypins** [comment]
I don't have a lot of experience with Java, but I'd also like to help if I can.

**Pull request merged**